spec/lib/hamster/list/each_spec.rb

```diff
@@ -32,10 +32,10 @@
         context "without a block" do
           it "returns an Enumerator" do
             list.each.class.should be(Enumerator)
-            list.each.to_list.should eql(list)
+            Hamster::List[*list.each].should eql(list)
```

Comment:
it is, but there's no reason for this spec to depend on `core_ext` (which it did implicitly before).